src/CommandHandler.ts

```diff
@@ -241,7 +241,16 @@ export namespace CommandHandler {
 		await client.login(options.token);
 		prefixes.push(`<@${client?.user?.id}> `);
 		prefixes.push(`<@!${client?.user?.id}> `);
-		owners.push((await client.fetchApplication()).owner?.id ?? '');
+		const appOwner = (await client.fetchApplication()).owner
+		if (appOwner) {
+			if (appOwner instanceof Team) {
+				for(const member of appOwner.members.map(m => m)) {
```

Comment:
Why the `.map(m => m)` ?

Reply:
I forgot the .array() method work too ^^'
And it's because it's a collection so I cant use it as only member, it give me [string, TeamMember]

Reply:
So use `.array()` ^^

Reply:
Just modified this and yes because Github was broken, I changed this on my fork but it's not linked

Reply:
You have to `checkout` to the PR on this repo to commit so

`git fetch origin pull/77/head:$LOCAL_BRANCH_NAME
git checkout $LOCAL_BRANCH_NAME`

`$LOCAL_BRANCH_NAME` has to be a string without spaces to set the branch name in your local repo.

Reply:
It can find it